driver/src/error.rs

```diff
@@ -100,6 +104,24 @@ impl From<PriceFindingError> for DriverError {
     }
 }
 
+impl From<ethsign::Error> for DriverError {
+    fn from(error: ethsign::Error) -> Self {
+        DriverError::new(&error.to_string(), ErrorKind::SigningError)
+    }
+}
+
+impl From<ethcontract::errors::DeployError> for DriverError {
+    fn from(error: ethcontract::errors::DeployError) -> Self {
+        DriverError::new(&error.to_string(), ErrorKind::ContractDeployedError)
+    }
+}
+
+impl From<ethcontract::errors::ExecutionError> for DriverError {
+    fn from(error: ethcontract::errors::ExecutionError) -> Self {
+        DriverError::new(&error.to_string(), ErrorKind::ContractExecutionError)
+    }
+}
+
```

Comment:
Nice to see some more refined Errors! Is this an added benefit of using `ethcontracts-rs`?

Reply:
Its more that `ethcontracs-rs` adds new errors that need to be accounted for.